src/app/fdctl/main1.c

```diff
@@ -130,6 +132,7 @@ fdctl_boot( int *        pargc,
   config.log.log_fd = fd_log_private_logfile_fd();
   fd_shmem_private_boot( pargc, pargv );;
   fd_tile_private_boot( 0, NULL );
+  fd_flamenco_boot( pargc, pargv );
```

Comment:
Can we remove this for now? Not sure it's needed and it's glibc specific.

Reply:
The yaml dump depends on this for printing public keys

Reply:
Can you make it not depend on this?

Reply:
That would be a @ripatel-fd thing

Reply:
It's harmless in the non-gcc case

Reply:
It's not going to dump it properly? This dependency is annoying and I'd prefer to not have it sneak in here when it's not really needed yet.